### PR TITLE
fix(select): The dropdown list is not changing colours when High Contrast Mode is enabled.

### PR DESCRIPTION
**Pull request by creador-dev**

### CODE REVIEW

File: assets/js/showcase/globals/admin-bar.js [hunk 1]
```diff
@@ -57,10 +57,12 @@
 					button.removeClass( 'active' );
 					button.attr( 'title', 'Activate color accessibility' );
 					suiWrap.removeClass( 'sui-color-accessible' );
+					body.removeClass( 'sui-select-accessible' );
```
Comment (iamleigh) [inline assets/js/showcase/globals/admin-bar.js:60]: @creador-dev this is not a viable solution. This file is to simulate some actions on the showcase but won't produce any real fix when the SUI package gets installed in a plugin. Please, move this action to be part of the library, like inside `select2.js` file.

File: assets/scss/shared-ui/_color-accessibility.scss [hunk 2]
```diff
@@ -577,3 +577,11 @@
 		}
 	}
 }
+
+@include body-class(false) {
+	&.sui-select-accessible {
+		.select2-results__option--selected{
+			background-color: $accessible-dark !important;
```
Comment (iamleigh) [inline assets/scss/shared-ui/_color-accessibility.scss:584]: @creador-dev, more elements need to be highlighted with black or white, depending on the case, for the component to be appropriately accessible—for example, the dropdown border and unselected items.

**Review by iamleigh — CHANGES_REQUESTED** [review]

File: assets/js/showcase/globals/admin-bar.js (hunk 1 above)
Comment (creador-dev) [reply assets/js/showcase/globals/admin-bar.js:60]: We have to use this solution to show select 2 style in showcase. I have added js in select2.js for sui accessible color to work in plugin.

File: assets/scss/shared-ui/_color-accessibility.scss (hunk 2 above)
Comment (creador-dev) [reply assets/scss/shared-ui/_color-accessibility.scss:593]: i have added the changes accrodingly.

**Review by iamleigh — APPROVED** [review]

**Pull request merged**